src/jarabe/journal/listview.py

```diff
@@ -711,6 +716,17 @@ def __drag_data_get_cb(self, widget, context, selection, info, time):
         model = self.tree_view.get_model()
         model.do_drag_data_get(path, selection)
 
+    def __button_press_event_cb(self, widget, event):
```

Comment:
The previously used `tree_view.enable_model_drag_source` returned the whole row to be drag. It didn't need any icon or something, the whole row would be seen lifted.

But as for using the `tree_view.drag_source_set` sets up a widget on which drag operation will start but icon wont be set automatically 

For starting the drag operation here we are using the `button-press-event`. 

So I have created another function for the `button-press-event` and also to set up an icon for indicating the drag.